js/html-forms_Finding-a11y-errors/index.html

```diff
@@ -11,21 +11,27 @@
   </head>
   <body>
     <section>
-      <h1 class="title">Understanding A11y in Forms</h1>
+      <h1 class="title" id="title">Understanding A11y in Forms</h1>
```

Comment:
Not sure if the id attribute has any influence for accessibility, let's ask Sven.